src/Common/Http/Client/ClientInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * SocialConnect project
+ * @author: Patsura Dmitry https://github.com/ovr <talk@dmtry.me>
+ */
+
+namespace SocialConnect\Common\Http\Client;
+
+use SocialConnect\Common\Http\Request;
+
+interface ClientInterface
```

Comment:
Instead of making your own interface I would suggest using the interface from [PSR-18](https://www.php-fig.org/psr/psr-18/) or extend it if you want to add extra methods.

Reply:
This code is moved from https://github.com/SocialConnect/common
Related to PSR-18, i will review and play with it a little bit before moving to it

Thanks